.github/workflows/expo-compatibility.yml

```diff
@@ -0,0 +1,71 @@
+name: Expo Compatibility Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test-expo-compatibility:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        expo-version: ['54']
```

Comment:
what if the end-user didn't specify a version, could we test against lastest, latest -1, and canary?